### PR TITLE
Update edgehub.md

### PR DESCRIPTION
**Pull request by didier-durand**
Hello,
Proposing to add links to beehive framework to allow readers to get details about it: I had to look it up to better understand.
Didier

### CODE REVIEW

**Comment by daixiang0** [comment]
In fact, we use `kubeedge/beehive`.

File: docs/components/edge/edgehub.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ A keep-alive message or heartbeat is sent to cloudHub after every heartbeatPerio
 
 - The main responsibility of publishing client info is to inform the other groups or modules regarding the status of connection to the cloud.
 
-- It sends a beehive message to all groups (namely metaGroup, twinGroup and busGroup), informing them whether cloud is connected or disconnected.
+- It sends a [beehive](https://github.com/kandoo/beehive) message to all groups (namely metaGroup, twinGroup and busGroup), informing them whether cloud is connected or disconnected.
```
Comment (fisherxu) [inline docs/components/edge/edgehub.md:28]:
```suggestion
- It sends a [beehive](https://github.com/kubeedge/beehive) message to all groups (namely metaGroup, twinGroup and busGroup), informing them whether cloud is connected or disconnected.
```

File: docs/components/edge/edgehub.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ The major steps involved in this process are as follows :-
 ## Route To Edge
 
 The main responsibility of route to edge is to receive messages from the cloud (through the websocket connection) and
-send them to the required groups through the beehive framework.
+send them to the required groups through the [beehive framework](https://github.com/kandoo/beehive).
```
Comment (fisherxu) [inline docs/components/edge/edgehub.md:53]:
```suggestion
send them to the required groups through the [beehive framework](https://github.com/kubeedge/beehive).
```
:)

**Comment by didier-durand** [comment]
@fisherxu : thanks for your review. I applied your suggestions to the PR. I guess that it can be merged now. Best, Didier

**Comment by didier-durand** [comment]
@fisherxu : Github now complains on invalid commit messages due to the @ with your name which is not allowed in commit messages. Can you fix that or do you prefer me to close this one and reopen a fresh one with exact same changes.

Sorry for the disturbance!

Didier

**Comment by daixiang0** [comment]
@didier-durand, please refer to [doc](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) to update.

**Comment by fisherxu** [comment]
@didier-durand  Please just squash the commits :)

**Comment by didier-durand** [comment]
@fisherxu : is squash merging as per https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-commit-squashing-for-pull-requests   allowed on this repo? Then, I will squash my commits. Thanks! Didier

**Comment by fisherxu** [comment]
@didier-durand The merge bot don't use `squash merging` here, so you need to squash them.

**Comment by didier-durand** [comment]
commits now squashed.

**Comment by fisherxu** [comment]
> commits now squashed.

Squash to 1 commit :)

**Comment by didier-durand** [comment]
Hi there,
I couldn't squash my commits properly on this one
So, simpler to re-open a fresh one: https://github.com/kubeedge/kubeedge/pull/2332
closing
Sorry for the noise!
didier

**Pull request closed without merging**